docs/questionRuntime/index.md

```diff
@@ -28,6 +28,8 @@ The quickest way to add custom libraries is to install them directly to your cou
 
 After these steps, you should be able to `import` the library as normal in your `server.py` files.
 
+**Note:** The local installation may generate compiled files that are commonly ignored in `.gitignore` (for example, `*.so` files). These files are required for the usage of the installed package, so make sure these files are committed within Git if you encounter issues. Ignored files can be viewed with `git status --ignored`.
```

Comment:
Can you provide an explicit example? Where are these .so files going to go? What prompted this note (can you give an example library this happens for?)

Reply:
These usually go into serverFilesCourse (building off of the instructions above) and there's discussion in the original issue. I don't think there's a good simple common example for this, since for commonly used libraries with compiled components, we usually add them to the base image to avoid this annoying installation step.